go.mod

```diff
@@ -83,19 +83,19 @@ require (
 	gopkg.in/square/go-jose.v2 v2.6.0
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1
-	helm.sh/helm/v3 v3.10.0
+	helm.sh/helm/v3 v3.10.3
 	istio.io/api v0.0.0-20221216020910-079613a3cd3b
 	istio.io/client-go v1.16.2-0.20221216021310-1000b11d0c2e
 	istio.io/pkg v0.0.0-20221107183613-574f8d141535
-	k8s.io/api v0.25.1
-	k8s.io/apiextensions-apiserver v0.25.0
-	k8s.io/apimachinery v0.25.1
-	k8s.io/apiserver v0.25.1
-	k8s.io/cli-runtime v0.25.1
-	k8s.io/client-go v0.25.1
+	k8s.io/api v0.25.2
+	k8s.io/apiextensions-apiserver v0.25.2
```

Comment:
Why are these changed? we try to minimize dependency changes in patch releases.

Reply:
they are dependencies of the helm fix

Reply:
ill test with the 25.1 though, i think this is a bigger issue for the 1.15 release changes

Reply:
i checked and this includes the CVE fix in k8s as well https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md#changelog-since-v1254

Reply:
https://github.com/kubernetes/client-go/releases/tag/v0.25.2

Reply:
looks like i should maybe update it to 25.4 to include the fix

Reply:
I'm not terribly opposed but I don't see any Kubernetes CVEs?

Reply:
https://github.com/kubernetes/kubernetes/pull/114320

Reply:
Why do we need to update k8s if the CVE is in golang.org/x/net? can we just update golang.org/x/net

Reply:
<img width="1143" alt="Screenshot 2022-12-28 at 3 33 24 PM" src="https://user-images.githubusercontent.com/22355055/209874458-2933a0bf-8a2e-48e3-9e89-41ac97096b28.png">
by modifying just the helm version from v3.10.0 to v3.10.3 and running `make format-go` its pulling in those versions

Reply:
@howardjohn does this address your concern?

Reply:
Not really? It may mean my concern may not be fix-able though and the value of appeasing scanners outweighs the concern

Reply:
Agree that the changes seem to match what is done by doing a `go get helm.sh/helm/v3@v3.10.3` or updating the single value in the go.mod and doing a `go mod tidy`, the later being required by our tooling.